### PR TITLE
OCM-4601 | chore: bump api-model to v0.0.332

### PR DESCRIPTION
**Pull request by gdbranco**
Related issue: OCM-4601

### CODE REVIEW

**Review by ciaranRoche — APPROVED** [review]
/lgtm

File: CHANGES.md [hunk 1]
```diff
@@ -3,6 +3,11 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.380
+- Update model version v0.0.332
+  - Add `AdditionalInfraSecurityGroupIds` to `AWS` type
+  - Add `AdditionalControlPlaneSecurityGroupIds` to `AWS` type
```
Comment (cristianoveiga) [inline CHANGES.md:9]: I'm seeing changes related to https://github.com/openshift-online/ocm-api-model/pull/854, should we add them here?

Comment (ciaranRoche) [reply CHANGES.md:9]: This ^^^ we have got out of sync between the version on the api model also. We need a new tag in the api-model and we also need to update the changes here

Comment (gdbranco) [reply CHANGES.md:9]: @ciaranRoche 332 is the latest, but I didn't notice that 331 hadn't been created in time, so I created only the tag later. Over here it is just missing to include the changes of the bump

Comment (gdbranco) [reply CHANGES.md:9]: https://github.com/openshift-online/ocm-api-model/releases

Fixed

**Pull request merged**